ibis/backends/snowflake/__init__.py

```diff
@@ -671,6 +672,95 @@ def drop_table(
         with self.begin() as con:
             con.exec_driver_sql(drop_stmt)
 
+    def read_csv(
+        self, path: str | Path, table_name: str | None = None, **kwargs: Any
+    ) -> ir.Table:
+        """Register a CSV file as a table in the Snowflake backend.
+
+        Parameters
+        ----------
+        path
+            Path to the CSV file
+        table_name
+            Optional name for the table; if not passed, a random name will be generated
+        kwargs
+            Snowflake-specific file format configuration arguments. See the documentation for
+            the full list of options: https://docs.snowflake.com/en/sql-reference/sql/create-file-format#type-csv
+
+        Returns
+        -------
+        Table
+            The table that was read from the CSV file
+        """
+        stage = ibis.util.gen_name("stage")
+        file_format = ibis.util.gen_name("format")
+        threads = min((os.cpu_count() or 2) // 2, 99)
```

Comment:
Why is the exact 99 upper limit?

Reply:
Who knows? It's a limitation of snowflake https://docs.snowflake.com/en/sql-reference/sql/put#optional-parameters

Reply:
Added a comment.